src/mgmt/services/manager/project/api/nodes.py

```diff
@@ -87,6 +87,8 @@ def create_droplet(name, ip, mac, itf, network, zgc_id):
 @nodes_blueprint.route('/nodes', methods=['GET', 'POST'])
 def all_nodes():
     if request.method == 'POST':
+        print('Start to make one node, and two droplets for this node.', flush=True)
```

Comment:
can we use logger or it's not working? That helps to control log level and where to output

Reply:
fixed.